### PR TITLE
New 'integrations' section to the docs with instructions for CodiMD

### PR DESCRIPTION
**Pull request by SamuAlfageme**
- Include a disclaimer on the kubernetes/wopiserver docs to mention
the (websocket) requirements when the cluster is running behind a
reverse proxy

### CODE REVIEW

**Comment by Daniel-WWU-IT** [comment]
Looks very good to me. Just two edge cases that caused problems for us and might be noteworthy:
- The `/wopib` endpoint can collide with the `/wopi` endpoint depending on the ingress configuration (if not using nginx, for example)
- You should give the users the ability to either use MariaDB or Postgresql for CodiMD. In our case, the latter did not work at all.

File: docs/content/en/docs/Integrations/codimd.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+## Deploying CodiMD through the `sciencemesh/meshapps` chart
+
+To deploy the application, the `sciencemesh/meshapps` umbrella chart already provides a handful of defaults for a lightweight and "_stateless_" CodiMD installation:
+
+- Based on a patched `codimd` container image (`gitlab-registry.cern.ch/authoring/notes/codimd:cernbox-integration`) with some extra features to enable this workflow.
+- Enables anonymous access, viewing and editing.
+- Disables some defaults from the vanila CodiMD installation:
+  - The registration and authentication mechanisms.
+  - The image store volume and PostreSQL disk persistence, as we will rely on the IOP Storage Providers to read and write all the content, including attached pictures.
```
Comment (glpatcern) [inline docs/content/en/docs/Integrations/codimd.md:55]: To be exact, CodiMD still has its own store + DB, but it is ephemeral (and in the future we want to deploy some automatic garbage collection for the image store, e.g. after one week). The sentence `we will rely...` is of course true.

File: docs/content/en/docs/Integrations/codimd.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+After this file is defined, it's just a matter of upgrading the chart while reusing any previous values, as follows:
+
+```bash
+helm upgrade -i meshapps sciencemesh/meshapps --reuse-values -f codimd.yaml
+```
+
+### Registering the MIME-types and custom file extensions on your Storage Provider
+
+There is three extra steps left for the IOP to integrate with this application. These will require modifying the Revad configuration to identify the files that will be opened by CodiMD:
```
Comment (glpatcern) [inline docs/content/en/docs/Integrations/codimd.md:100]: `is` -> `are`

File: docs/content/en/docs/Integrations/codimd.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+2. Optionally, the `wopibridge` URL might be inserted as part of the `appprovider` settings on the gateway; to generate a shorter and user-friendlier URL when opening from the reva cli.
+
+```toml
+[grpc.services.appprovider]
+wopibridgeurl = "https://<hostname.domain.tld>/wopib"
+```
+
+
+3. Add a mapping between the custom `.zmd` file extension, used to package markdown files together with uploaded images, and the `application/compressed-markdown` MIME-type used to identify this files. This needs to be done on all the running `storageprovider` services: whether using a [standalone](../../iop/deployment/kubernetes) deployment or one based in [multiple, decoupled storage providers](../../iop/deployment/kubernetes/providers).
```
Comment (glpatcern) [inline docs/content/en/docs/Integrations/codimd.md:118]: `this files` -> `these files`

**Review by glpatcern — CHANGES_REQUESTED** [review]
Looks very good Samuel, thanks a lot. I suggest just a few changes on top of what Daniel suggested (MariDB as a different option for CodiMD).

File: docs/content/en/docs/Integrations/codimd.md (hunk 1 above)
Comment (SamuAlfageme) [reply docs/content/en/docs/Integrations/codimd.md]: You're absolutely right. I've just poured some more clarity into that sentence. Thx @glpatcern 🙌

**Review by glpatcern — APPROVED** [review]
You caught a few extra fixes I did not spot :) all good for me.

**Pull request merged**